apps/portal/src/app/contracts/modular-contracts/get-started/create-core-contract/page.mdx

```diff
@@ -29,152 +34,196 @@ forge remappings > remappings.txt
 <Step title="Create a Core Contract">
    Create a new file in the `src` folder called `CounterCore.sol`, and inherit the `Core` contract.
 
-   ```solidity
-   // SPDX-License-Identifier: UNLICENSED
-   pragma solidity ^0.8.20;
+```solidity
+ // SPDX-License-Identifier: MIT
+ pragma solidity ^0.8.20;
 
-   import {Core} from "modular-contracts/src/Core.sol";
+ import {Core} from "@thirdweb-dev/src/Core.sol";
+ import {BeforeIncrementCallback} from "./interface/BeforeIncrementCallback.sol";
 
-   contract CounterCore is Core {
+ contract CounterCore is Core {
+     constructor(address owner) {
+         _initializeOwner(owner);
+     }
+ }
 
-        constructor(address owner) {
-            _initializeOwner(owner);
-        }
+```
+
+> **Note**  
+> The `Core` contract is the base contract that needs to be inherited for this contract to be recognized as a core contract.
+
+</Step>
 
-   }
-   ```
+<Step title="Set Get Supported Callback Function">
+Implement the `getSupportedCallbackFunctions` function. The Core contract is abstract because this function is not implemented. To avoid compilation errors, declare the function with an empty body for now.
 
-   > **Note**  
-   > The `Core` contract is the base contract that needs to be inherited for this contract to be recognized as a core contract.
+    ```solidity
+        // SPDX-License-Identifier: MIT
+        pragma solidity ^0.8.20;
+
+        import {Core} from "@thirdweb-dev/src/Core.sol";
+
+        contract CounterCore is Core {
+            constructor(address owner) {
+                _initializeOwner(owner);
+            }
+
+            function getSupportedCallbackFunctions()
+                public
+                pure
+                override
+                returns (SupportedCallbackFunction[] memory supportedCallbackFunctions)
+            {}
+        }
+    ```
 
 </Step>
 
 <Step title="Set Up Increment Function">
    Define a function to increment a counter.
 
-   ```solidity
-   // SPDX-License-Identifier: UNLICENSED
-   pragma solidity ^0.8.20;
+```solidity
+ // SPDX-License-Identifier: MIT
+ pragma solidity ^0.8.20;
 
-   import {Core} from "modular-contracts/src/Core.sol";
+ import {Core} from "@thirdweb-dev/src/Core.sol";
 
-   contract CounterCore is Core {
-       uint256 public count;
+ contract CounterCore is Core {
+     int256 public count;
```

Comment:
leave this as a `uint256`. There's a big difference between uint256 and int256. If there isn't a use case for negative numbers, it's best practice to default to uint256